tljh/installer.py

```diff
@@ -217,14 +217,14 @@ def ensure_jupyterhub_package(prefix):
     ])
 
     conda.ensure_pip_packages(prefix, [
-        'jupyterhub==1.0.0',
+        'jupyterhub==1.1.0',
         'jupyterhub-dummyauthenticator==0.3.1',
         'jupyterhub-systemdspawner==0.13',
-        'jupyterhub-firstuseauthenticator==0.13.0',
-        'jupyterhub-nativeauthenticator==0.0.4',
-        'jupyterhub-ldapauthenticator==1.2.2',
+        'jupyterhub-firstuseauthenticator==0.14.1',
+        'jupyterhub-nativeauthenticator==0.0.5',
+        'jupyterhub-ldapauthenticator==1.3.0',
         'jupyterhub-tmpauthenticator==0.6',
-        'oauthenticator==0.10.0',
+        'oauthenticator==0.11.0',
```

Comment:
As of OAuthenticator 0.11, there's no AWSCognito authenticator ([changelog ref](https://github.com/jupyterhub/oauthenticator/blob/master/docs/source/changelog.md#0110---2020-01-30)).

However, TLJH has a [section describing how to use AWSCognito](http://tljh.jupyter.org/en/latest/howto/auth/awscognito.html). If we update oauthenticator, that section should be removed or adapted so that it uses GenericOAuthenticator instead.

I can open up an issue after this PR gets merged and deal with it later on. What do you think @jtpio ? :eyes:

Reply:
Good catch :+1: 

We could also keep `oauthenticator==0.10.0` for now, and still open the issue?

Reply:
Sure :sunny: